common/modifiers/heritage_modifiers_events.txt

```diff
@@ -871,8 +871,8 @@ tibetan_empire_modifier = {
 }
 
 baratanac_modifier = {

```

Comment:
undo all the changes in this file, they are overwriting changes i already made here